zerver/lib/email_notifications.py

```diff
@@ -111,7 +112,9 @@ def make_emoji_img_elem(emoji_span_elem: lxml.html.HtmlElement) -> Dict[str, Any
         emoji_code = match.group("emoji_code")
         emoji_name = emoji_span_elem.get("title")
         alt_code = emoji_span_elem.text
-        image_url = base_url + f"/static/generated/emoji/images-{emojiset}-64/{emoji_code}.png"
+        image_url = staticfiles_storage.url(
+            f"generated/emoji/images-{emojiset}-64/{emoji_code}.png"
+        )
```

Comment:
I think this part in particular is going to mean that the emoji in emails will break if the Zulip server is ever reinstalled -- since we'll have every different hash-version of the URLs for emoji from every past version that has been deployed on that installation present in emails in users' inboxes. 

That outcome is untenable, so I see basically two options: We could try to migrate to converting emoji elements in messages back into unicode emoji for this code path (which would mean you'd get your email client's emoji set rather than the emojiset you have configured in Zulip, which has both pluses and minuses), or we can stick with unhashed URLs here? 

(Or is this unhashed? Regardless, we probably want a long comment at this spot explaining this detail, since it's a land mine)

Reply:
This is unhashed, because as noted [above](https://github.com/zulip/zulip/pull/24286#discussion_r1096459990) we don't actually do any hashing in Django.

I've replaced with a big comment which explains why we're just using a `settings.STATIC_URL` prefix and not anything smarter.